anchore_engine/clients/grype_wrapper.py

```diff
@@ -231,32 +227,98 @@ def update_grype_db(grype_db_archive_local_file_location: str, version_name: str
             write_lock.release()
 
 
-def get_vulnerabilities(grype_sbom: str) -> json:
+def get_current_grype_db_metadata() -> json:
     """
-    Use grype to scan the provided sbom for vulnerabilites.
+    Return the json contents of the metadata file for the in-use version of grype db
     """
     global grype_db_dir
```

Comment:
I've considered refactoring this whole module into a class so these state variables, currently implemented as globals, can just become data on the class. Not in scope for this change, but still something I would like to do. @nightfurys What do you think? We'd probably have to make sure it was created as a singleton during startup so that all of the users of this facade get the same instance.